app/src/androidTest/java/com/arygm/quickfix/ui/search/ProfileResultsTest.kt

```diff
@@ -1,148 +1,116 @@
-  private lateinit var accountViewModel: AccountViewModel
-  private lateinit var categoryViewModel: CategoryViewModel
-  private lateinit var navigationActionsRoot: NavigationActions
-  private lateinit var workerRepository: ProfileRepository
-  private lateinit var workerViewModel: ProfileViewModel
-
   @get:Rule val composeTestRule = createComposeRule()
 
+  private lateinit var accountViewModel: AccountViewModel
+  private lateinit var searchViewModel: SearchViewModel
+
   @Before
   fun setup() {
-    navigationActions = mock(NavigationActions::class.java)
-    navigationActionsRoot = mock(NavigationActions::class.java)
-    workerProfileRepo = mockk(relaxed = true)
-    categoryRepo = mockk(relaxed = true)
-    accountRepositoryFirestore = mock(AccountRepositoryFirestore::class.java)
-    searchViewModel = SearchViewModel(workerProfileRepo)
-    categoryViewModel = CategoryViewModel(categoryRepo)
     accountViewModel = mockk(relaxed = true)
+    searchViewModel = mockk(relaxed = true)
 
-    workerViewModel = mockk(relaxed = true)
-
-    // Mock fetchProfileImageAsBitmap
-    every { workerViewModel.fetchProfileImageAsBitmap(any(), any(), any()) } answers
-        {
-          val onSuccess = arg<(Bitmap) -> Unit>(1)
-          // Provide a dummy bitmap here (e.g. a solid color bitmap or decode from resources)
-          val dummyBitmap = Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888)
-          onSuccess(dummyBitmap) // Simulate success callback
-        }
-
-    // Mock fetchBannerImageAsBitmap
-    every { workerViewModel.fetchBannerImageAsBitmap(any(), any(), any()) } answers
-        {
-          val onSuccess = arg<(Bitmap) -> Unit>(1)
-          // Provide another dummy bitmap
-          val dummyBitmap = Bitmap.createBitmap(20, 20, Bitmap.Config.ARGB_8888)
-          onSuccess(dummyBitmap) // Simulate success callback
-        }
-  }
+    // Mock calculateDistance to return a fixed distance
+    every { searchViewModel.calculateDistance(any(), any(), any(), any()) } returns 10.0
 
-  @Test
-  fun profileContent_displaysWorkerProfiles() {
-    // Set up test data
-    val testProfiles =
-        listOf(
-            WorkerProfile(
-                uid = "worker0",
-                fieldOfWork = "Plumbing",
-                rating = 3.5,
-                reviews = ArrayDeque(),
-                location = null, // Simplify for the test
-                price = 49.0,
-            ),
-            WorkerProfile(
-                uid = "worker1",
-                fieldOfWork = "Electrical",
-                rating = 3.0,
-                reviews = ArrayDeque(),
-                location = null,
-                price = 59.0,
-            ))
-
-    // Mock the AccountViewModel to return sample account data
+    // Mock AccountViewModel fetchUserAccount
     every { accountViewModel.fetchUserAccount(any(), captureLambda()) } answers
         {
           val uid = firstArg<String>()
```

Comment:
Why was this test removed ?

Reply:
this test was not removed but updated to match the update of the `ProfileResults` composable. It was replaced by a new test that is testing exactly the same thing